ambry-router/src/test/java/com.github.ambry.router/GetManagerTest.java

```diff
@@ -112,7 +114,27 @@ public void testSimpleBlobGetSuccess()
   public void testCompositeBlobGetSuccess()
       throws Exception {
     router = getNonBlockingRouter();
-    setOperationParams(chunkSize * 6 + 11);
+    setOperationParams(chunkSize * 6 + 11, null);
+    String blobId = router.putBlob(putBlobProperties, putUserMetadata, putChannel).get();
```

Comment:
any chance you can put these lines away into a function? (Considering that the next test seems to have the same lines)